ERROR  Cannot find module 'react/jsx-runtime' : building "Content" with Solidjs

Error:
```
WXT 0.16.8                                                                                   10:17:04 PM
ℹ Generating types...                                                                       10:17:06 PM

 ERROR  Cannot find module 'react/jsx-runtime'                                               10:17:06 PM  
Require stack:
- D:\EXTENSION\chrome-nuxt\entrypoints\content\index.jsx

  Require stack:
  - entrypoints\content\index.jsx
  at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
  at Function.resolve (node:internal/modules/cjs/helpers:127:19)
  at _resolve (node_modules\.pnpm\jiti@1.21.0\node_modules\jiti\dist\jiti.js:1:251148)
  at jiti (node_modules\.pnpm\jiti@1.21.0\node_modules\jiti\dist\jiti.js:1:253746)
  at D:/EXTENSION/chrome-nuxt/entrypoints/content/index.jsx:23:26
  at evalModule (node_modules\.pnpm\jiti@1.21.0\node_modules\jiti\dist\jiti.js:1:256443)
  at jiti (node_modules\.pnpm\jiti@1.21.0\node_modules\jiti\dist\jiti.js:1:254371)
  at importEntrypointFile (/D:/EXTENSION/chrome-nuxt/node_modules/.pnpm/wxt@0.16.8/node_modules/wxt/dist/cli.js:2015:23)

```

`App.jsx`
```
import { createSignal } from "solid-js";

export default () => {
  const [count, setCount] = createSignal(1);
  const increment = () => setCount((count) => count + 1);

  return (
    <div>
      <p>This is SOLID. {count}</p>
      <button onClick={increment}>Increment</button>
    </div>
  );
};

```

`Index.jsx`
```

import App from './App.jsx';
import {render } from 'solid-js/web';

export default defineContentScript({
  matches: ['<all_urls>'],
  // 2. Set cssInjectionMode
  cssInjectionMode: 'ui',

  async main(ctx) {
    // 3. Define your UI
    const ui = await createShadowRootUi(ctx, {
      name: 'example-ui',
      position: 'inline',
      onMount: (container) => {
        // Render your app to the UI container
        const unmount = render(() => <App />, container);
      },
      onRemove: (unmount) => {
        // Unmount the app when the UI is removed
        unmount?.();
      },
    });

    // 4. Mount the UI
    ui.mount();
  },
});
```

--- a/src/core/utils/building/import-entrypoint.ts
+++ b/src/core/utils/building/import-entrypoint.ts
@@ -105,6 +105,13 @@ function getEsbuildOptions(opts: JitiTransformOptions): TransformOptions {
   return {
     format: 'cjs',
     loader: isJsx ? 'tsx' : 'ts',
-    jsx: isJsx ? 'automatic' : undefined,
+    ...(isJsx
+      ? {
+          // `h` and `Fragment` are undefined, but that's OK because JSX is never evaluated while
+          // grabbing the entrypoint's options.
+          jsxFactory: 'h',
+          jsxFragment: 'Fragment',
+        }
+      : undefined),
   };
 }

